### ISSUE TITLE
Wind Turbines turning into Hydroponic farms

### ISSUE THREAD

**Issue by qburt** [RavienGaming/Reforged-Eden-2#337]
### Reforged Eden 2 Build Number

22

### Game mode

Survival, Single Player, Co-op

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

When wind turbines are placed, they eventually turn into hydroponic farms.  The time it takes to change is variable.  I tried this on my survival server, in single player and in hosted co-op.  

I have:
validated install files
Reinstalled the game
Removed and reinstalled RE2
Deleted and reinstalled the server files
![windturbinebaseT2 ID](https://github.com/user-attachments/assets/a7f18910-d4c6-402e-9532-1206a91738a6)
![windturbineIDLE ID](https://github.com/user-attachments/assets/21537124-562f-444a-a283-b30d1f9295e1)
![windturbineturnedhydro ID](https://github.com/user-attachments/assets/16712e10-19fc-414f-a9db-9038fcbc4ecd)
![RE2 Build-22](https://github.com/user-attachments/assets/f24200c5-0aa5-4a73-ba3e-6d8a24ebbc8e)
![windturbine(active) ID](https://github.com/user-attachments/assets/acc9e001-64a9-4140-8630-0befa39ffb37)
![windturbine](https://github.com/user-attachments/assets/267529cc-3bb1-4065-83ba-a189267e49a4)
![windturbineAllTurned ID](https://github.com/user-attachments/assets/e9fd9476-cd1c-4101-8865-26eb235a15d0)
![windturbinebase ID](https://github.com/user-attachments/assets/bf45189f-ddf4-4e31-8362-a08aa3409a78)

Both the Wind Turbine and the advanced wind turbine to include the base turn into hydroponic farms.

### Playfield Name

Tehlimine

### Playfield Class

Planet

### Star Name

Voloflas

### Star Class

F

### Mission Name

N/A

### Mission Objective

N/A

### Game Seed

7715557

**Comment by RavienGaming** [comment]
Duplicate of #10

**Comment by shockr** [comment]
TLDR: misconfiguration of gameoptions.yaml causes defaults to apply, including DecayTime which causes the turbines to decay into hydroponics.

Ok, so I think I've figured it out.
The symptoms were: Windturbines were turning into fruit hydros, then into empty hydros.
At first I thought a weather change was causing it, then I started timing it and it always happened within 4 minutes. However it wasn't until I checked 'gopt' that I noticed all the config settings were defaulted. Including 'DecayTime'.
The reason for this was actually an error introduced into the gameoptions.yaml. Yaml is notorious for being very brittle. A tab instead of a space etc..
Replacing the gameoptions with one from a backup, the game settings took hold after rebooting the dedicated server.
With the DecayTime now at 0, the windturbines have remained unfruited.

**Comment by RavienGaming** [comment]
Duplicate of #10
